src/js/preshuffle.s

```diff
@@ -781,15 +781,19 @@ MaybeSetCheckpointActual:
   .byte $20,$20,$1f     ;  _ _ |
 
 .org $bb1b
-  .byte $20             ;  _
-.org $bb1f ; clear out the experience count thats there initially
-.repeat 16
+  .byte $82             ; Ey
+; clear out the experience count thats there initially
+; and make room for something like enemy name
+.repeat 11
```

Comment:
What do you think about putting the number on the left and allowing the name to go more in the bottom-middle?  I feel like that gives a better separation between the player's stats and the enemy stats.